build/broad_sanger/05b_separate_datasets.py

```diff
@@ -70,14 +70,16 @@ def main():
 
         #One by one, filter other Omics files, write to file, delete from mem.
         for omics in omics_datatypes:
-            omics_filename_in = f"broad_sanger_{omics}.csv"
```

Comment:
@jjacobson95 here is where we read in the separate files, no need to filter on source. can you please rever these back